shuup/admin/template_helpers/shuup_admin.py

```diff
@@ -181,3 +181,9 @@ def get_admin_shop(context):
 @contextfunction
 def is_multishop_enabled(context):
     return settings.SHUUP_ENABLE_MULTIPLE_SHOPS is True
+
+
+@contextfunction
+def get_current_language(context):
```

Comment:
You should have `{{ LANGUAGE_CODE }}` available at the template.

Reply:
This is not always available in admin. When importing the macro, if one forgot to import `with context`, that won't be available

Reply:
ok